web/src/components/molecules/Schema/ModelFieldList.tsx

```diff
@@ -10,17 +10,35 @@ import { Field } from "./types";
 export interface Props {
   className?: string;
   fields?: Field[];
+  handleFieldDelete: (fieldId: string) => Promise<void>;
+  handleFieldUpdateModalOpen: (field: Field) => void;
 }
 
-const ModelFieldList: React.FC<Props> = ({ fields }) => {
+const ModelFieldList: React.FC<Props> = ({
```

Comment:
This component is wrapping its return components in a fragment. Please be more careful and avoid unnecessary fragments